business_requirement_deliverable_cost/models/business.py

```diff
@@ -42,11 +42,13 @@ def product_id_change(self):
                 resource.business_requirement_deliverable_id.project_id
             if deliverable_project.pricelist_id and \
                     deliverable_project.partner_id and resource.uom_id:
+                pricelist = \
```

Comment:
Move this before the ```if``` to use it in the condition. It should use get_estimation_pricelist (of the BR)